utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/JsonSchemaBasedParameter.scala

```diff
@@ -65,18 +75,45 @@ object JsonSchemaBasedParameter {
       }
     }
 
+    def toSchemaBasedParameterWithEmptySchemaSupport(
+        schema: Schema,
+        paramName: Option[ParameterName],
+        defaultValue: Option[Expression],
+        isRequired: Option[Boolean]
+    ): ValidatedNel[ProcessCompilationError, SchemaBasedParameter] = {
+      schema match {
+        case _: EmptySchema =>
+          // In a case of an empty JSON schema we want to present the user with Json editor and not standard SpEL one
+          Valid(
+            createJsonSinkSingleValueParameter(
```

Comment:
Let's copy this method instead of hacking the existing one. The new method won't have schema, defaultValue, isRequired and editor. Let's method name include "json editor" to explicitly show what's going on. If you already extracted a new "support" for schemaless case, I assume that  we don't even need to go through `withEmptySchemaSupport` -> `toSchemaBasedParameterWithEmptySchemaSupport` and do pattern matching on empty schema.